msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/PrivateMessagesTool.java

```diff
@@ -1446,16 +1446,32 @@ public String processPvtMsgForward() {
 		StringBuilder forwardedText = new StringBuilder();
 	    
 	    // populate replyToBody with the forwarded text
-		forwardedText.append(getResourceBundleString("pvt_msg_fwd_heading") + "<br /><br />" +
-	    	getResourceBundleString("pvt_msg_fwd_authby", new Object[] {formattedText.escapeHtml(pm.getAuthor(), false), formattedCreateDate}) +  "<br />" +
-	    	getResourceBundleString("pvt_msg_fwd_to", new Object[] {pm.getRecipientsAsText()}) + "<br />" +
-	    	getResourceBundleString("pvt_msg_fwd_subject", new Object[] {pm.getTitle()}) + "<br />" +
-	    	getResourceBundleString("pvt_msg_fwd_label", new Object[] {getDetailMsg().getLabel()}));
+		forwardedText.append(getResourceBundleString("pvt_msg_fwd_heading") + "<br /><br />");
+		forwardedText.append("<strong>");
+		forwardedText.append(getResourceBundleString("pvt_msg_fwd_authby"));
+		forwardedText.append("</strong>");
+		forwardedText.append(" " + formattedText.escapeHtml(pm.getAuthor(), false));
+		forwardedText.append(" (" + formattedCreateDate + ")<br />");
+		forwardedText.append("<strong>");
+		forwardedText.append(getResourceBundleString("pvt_msg_fwd_to"));
+		forwardedText.append("</strong>");
+		forwardedText.append(" " + pm.getRecipientsAsText() + "<br />");
+		forwardedText.append("<strong>");
+		forwardedText.append(getResourceBundleString("pvt_msg_fwd_subject"));
+		forwardedText.append("</strong>");
+		forwardedText.append(" " + pm.getTitle() + "<br />");
+		forwardedText.append("<strong>");
+		forwardedText.append(getResourceBundleString("pvt_msg_fwd_label"));
+		forwardedText.append("</strong>");
+		forwardedText.append(" " + getDetailMsg().getLabel());
 	    
 	    List attachList = getDetailMsg().getAttachList();
 	    if (CollectionUtils.isNotEmpty(attachList)) {
-	    	forwardedText.append(getResourceBundleString("pvt_msg_fwd_attachments") + "<br />");
-	    	forwardedText.append("<ul style=\"list-style-type:none;margin:0;padding:0;padding-left:0.5em;\">");
+	    	forwardedText.append("<div>");
+	    	forwardedText.append("<strong>");
+	    	forwardedText.append(getResourceBundleString("pvt_msg_fwd_attachments"));
+	    	forwardedText.append("</strong>");
+	    	forwardedText.append("<ul style=\"list-style-type:none;margin-bottom:1.0em;margin-left:0;margin-right:0;margin-top:0;padding-left:0.5em;padding:0\">");
```

Comment:
forwardedText.append("<ul class='list-unstyled ms-2 p-0 mb-3'>");

Reply:
@ottenhoff I don't quite understand the comment. I've tried to follow the existing format in the file. Tell me exactly, and I'll correct the issue. Thanks!

Reply:
it's just a suggestion to use bootstrap classes instead of inline styling (that can't be easily overridden)

not a big deal

Reply:
Thanks, I think your previous comment wasn't showing up for some reason.